app/views/tables/_form.html.erb

```diff
@@ -1,98 +1,109 @@
-        </td>
-        <td><%= f.number_field :prereg_price %> </td>
-      </tr>
+    <tr>
+      <td>
+        <%= f.label :premium, 'Is this a premium table?' %>
+        <div class="text-info">Premium tables are for fee only, at time of registration.</div>
+      </td>
+      <td>
+        <%= f.check_box :premium %>
+      </td>
+    </tr>
+    <!-- put a conditional here for javascript -->
+    <tr>
+      <td>
+        <%= f.label :prereg_price, 'What is the Early Bird or Prereg price for joining this table?' %>
+        <div>This will be used before <%= @event.prereg_ends.localtime.to_formatted_s(:long_ordinal) %></div>
+      </td>
+      <td><%= f.number_field :prereg_price %> </td>
+    </tr>
 
-      <tr>
-        <td>
-          <%= f.label :onsite_price, 'What is the Regular or Onsite price for joining this table?' %>
-          <div>This will be used after <%= @event.prereg_ends.localtime.to_formatted_s(:long_ordinal) %></div>
-        </td>
-        <td><%= f.number_field :onsite_price %> </td>
-      </tr>
+    <tr>
+      <td>
+        <%= f.label :onsite_price, 'What is the Regular or Onsite price for joining this table?' %>
+        <div>This will be used after <%= @event.prereg_ends.localtime.to_formatted_s(:long_ordinal) %></div>
+      </td>
+      <td><%= f.number_field :onsite_price %> </td>
+    </tr>
 
-      <tr>
-        <td><%= f.label :core, 'CORE table' %></td>
-        <td><%= f.check_box :core %></td>
-      </tr>
+    <tr>
+      <td><%= f.label :core, 'CORE table' %></td>
+      <td><%= f.check_box :core %></td>
+    </tr>
 
-      <tr>
-        <td><%= f.label :information, 'Additional information for players (character requirements, etc).' %></td>
-        <td><%= f.text_area :information %>
-      </tr>
+    <tr>
+      <td><%= f.label :information, 'Additional information for players (character requirements, etc).' %></td>
+      <td><%= f.text_area :information %>
+    </tr>
 
-      <tr>
-        <td><%= f.label :max_players %></td>
-        <td><%= f.number_field :max_players %></td>
-      </tr>
+    <tr>
+      <td>
+        <%= f.label :max_players, 'What is the maximum number of players?' %>
+        <div><em>(Typically 6 per physical table)</em></div>
+      </td>
+      <td><%= f.number_field :max_players %></td>
+    </tr>
 
-      <tr>
-        <td><%= f.label :gms_needed %></td>
-        <td><%= f.number_field :gms_needed %></td>
-      </tr>
-    </table>
+    <tr>
+      <td>
+        <%= f.label :gms_needed, 'How many GMs are needed for this table' %>
+        <div><em>(suggested 1 GM per 6 players)</em></div>
+      </td>
+      <td><%= f.number_field :gms_needed %></td>
+    </tr>
 
-    <div class="actions">
-      <%= f.submit 'Save Changes', :class => 'btn btn-primary' %>
-    </div>
+    <tr>
+      <td><%= f.label :gm_self_select, 'Can GMs select this table for themselves?' %></td>
+      <td><%= f.check_box :gm_self_select %></td>
```

Comment:
I presume this is something that can be set up to stop people from selecting themselves for our paid tables, etc

Reply:
Options!
:money_with_wings: